Bottom: abb11c53e0f101d8b1119d5374443185fa72c1b3
Top:    c12977ecfd6c8d6dabcddc84c03ad11f4eba9906
Author: Peter Clifton <peter@clifton-electronics.co.uk>
Date:   2016-12-04 19:24:24 +0000

Disable perturbation attempt during intersection


---

diff --git a/src/polygon1.c b/src/polygon1.c
index 3ef2e74..6f24bd1 100644
--- a/src/polygon1.c
+++ b/src/polygon1.c
@@ -1158,6 +1158,7 @@ seg_in_seg_arc_line (struct info *i, struct seg *s1, struct seg *s2)
 
   printf ("  Intersect count is %i\n", cnt);
 
+#if 0
   if (cnt == 0)
     {
       printf ("Trying with a perturbation\n");
@@ -1172,6 +1173,10 @@ seg_in_seg_arc_line (struct info *i, struct seg *s1, struct seg *s2)
 
       printf ("!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!! <<<<<<\n");
     }
+#endif
+
+  if (cnt == 0)
+    return 0;
 
   if (cnt == 2)
     {
